pkg/providers/vsphere/virtualmachine/snapshot.go

```diff
@@ -194,11 +232,11 @@ func GetSnapshotSize(vmCtx pkgctx.VirtualMachineContext, vmSnapshot *vimtypes.Ma
 
 	// Add the file key for the snapshot memory (.vmem) file if present.
 	if snapshot.MemoryKey != -1 {
-		vmCtx.Logger.V(5).Info("Adding memoryKey", "memoryKey", snapshot.MemoryKey)
+		logger.V(5).Info("Adding memoryKey", "memoryKey", snapshot.MemoryKey)
```

Comment:
Here and elsewhere in the change: I thought we said we will not have any log level below 4?

Reply:
mm these are very verbose log. Maybe just keep it as V(5) for now? I could still see many V(5) logging in the repo. Bryan also think it's better to keep V(5)